bionlp/taggers/rnn_feature/tagger_utils.py

```diff
@@ -123,6 +123,41 @@ def trim_tags(tagged_data):
                           for x, y in tagged_data[i]]
     return tagged_data
 
+def get_regular_or_capitalized_embeddings(w2i, i2w, mdl, params):
```

Comment:
Good point. I left it in since it corresponded with how the check was done before I wrote the new function, and I wanted to change that logic as little as possible, but it's ultimately pointless. I will remove it.